.github/workflows/release.yml

```diff
@@ -51,20 +51,20 @@ jobs:
       - run: pnpm -r publish --access=public
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
-  docs:
-    permissions:
-      contents: read
-      pages: write
-      id-token: write
-    needs: release
-    if: |
-      contains(fromJson(needs.release.outputs.paths_released), 'packages/access-client') ||
-      contains(fromJson(needs.release.outputs.paths_released), 'packages/blob-index') ||
-      contains(fromJson(needs.release.outputs.paths_released), 'packages/capabilities') ||
-      contains(fromJson(needs.release.outputs.paths_released), 'packages/did-mailto') ||
-      contains(fromJson(needs.release.outputs.paths_released), 'packages/upload-client') ||
-      contains(fromJson(needs.release.outputs.paths_released), 'packages/upload-api') ||
-      contains(fromJson(needs.release.outputs.paths_released), 'packages/filecoin-client') ||
-      contains(fromJson(needs.release.outputs.paths_released), 'packages/filecoin-api') ||
-      contains(fromJson(needs.release.outputs.paths_released), 'packages/w3up-client')
-    uses: './.github/workflows/reusable-deploy-docs.yml'
+  # docs:
```

Comment:
Wouldn't it be cleaner to delete it? One can always restore from history if needed